### PR TITLE
List order unexpected keyword

### PR DESCRIPTION
**Pull request by newoldtrader**
#20 
Adds order_placement_source and outstanding_hold_amount parameters to the Order class in orders.py

### CODE REVIEW

File: coinbaseadvanced/models/products.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ class ProductType(Enum):
 
     SPOT = "SPOT"
 
+Gran = {"ONE_MINUTE": 1, "FIVE_MINUTE": 5, "FIFTEEN_MINUTE": 15, "THIRTY_MINUTE": 30, "ONE_HOUR": 60, "TWO_HOUR": 120, "SIX_HOUR": 360, "ONE_DAY": 720}
```
Comment (KmiQ) [inline coinbaseadvanced/models/products.py:23]: I thought we agreed to change the naming for this.

**Comment by KmiQ** [comment]
This PR looks like a bit of a mix of things since we already were addressing the pagination issue in a different one. Ideally, I'd love it if we could keep the scope of this PR addressing the issue mentioned only. I already have a PR ready with a dedicated fix + unit tests. Gonna file it shortly.

**Comment by DKBrown14** [comment]
Yeah, sorry. I'm still getting used to git and managed to scramble my branches. 

I've got a bunch of other changes in my fork that I haven't pushed because I don't want to create more issues for you.

**Comment by KmiQ** [comment]
> Yeah, sorry. I'm still getting used to git and managed to scramble my branches.
> 
> I've got a bunch of other changes in my fork that I haven't pushed because I don't want to create more issues for you.

No worries. I just merged a fix (https://github.com/KmiQ/coinbase-advanced-python/pull/23) This can be helpful for you as an example about how to add unit tests etc... I'm gonna follow-up with more changes.

And please, continue creating issues and sharing feedback. It's much appreciated.

**Comment by KmiQ** [comment]
This PR can be closed since we already fixed all the issues this tried to solve.

**Comment by newoldtrader** [comment]
I agree.

**Pull request closed without merging**